homeassistant/components/monarchmoney/entity.py

```diff
@@ -0,0 +1,96 @@
+"""Monarch money entity definition."""
+
+from typing import Any
+
+from homeassistant.helpers.device_registry import DeviceEntryType, DeviceInfo
+from homeassistant.helpers.entity import EntityDescription
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import MonarchMoneyDataUpdateCoordinator
+
+
+class MonarchMoneyBaseEntity(CoordinatorEntity[MonarchMoneyDataUpdateCoordinator]):
```

Comment:
Why don't you create a base entity that just wraps the coordinator entity and has the has entity name?